tests/support/peerManager.ts

```diff
@@ -184,25 +186,44 @@ export class RadiclePeer {
     });
   }
 
-  public async startHttpd(port: number) {
+  public async startHttpd(port?: number) {
+    if (!port) {
+      port = await getPort();
+    }
+    this.#httpdBaseUrl = {
+      hostname: "0.0.0.0",
+      port,
+      scheme: "http",
+    };
     this.#httpdProcess = this.spawn("radicle-httpd", [
       "--listen",
-      `0.0.0.0:${port}`,
+      `${this.#httpdBaseUrl.hostname}:${this.#httpdBaseUrl.port}`,
     ]);
 
     await waitOn({
-      resources: [`tcp:127.0.0.1:${port}`],
+      resources: [
+        `tcp:${this.#httpdBaseUrl.hostname}:${this.#httpdBaseUrl.port}`,
+      ],
       timeout: 7000,
     });
+
+    return this.#httpdBaseUrl;
```

Comment:
I don’t think this return value is used anywhere. You can remove it

Reply:
yeah is a leftover probably doesn't need it